### DISCUSSION TITLE
[Gauge] Failed to start gauge API: Timed out connecting to python

### DISCUSSION THREAD

**Discussion by aratmz** [getgauge/gauge#1934] [General]
Been running into this issue for the past 2 weeks now

**Comment by aratmz** [comment]
`08-03-2021 15:42:39.185 [Gauge] [DEBUG] Created gauge_screenshots_dir at /Users/xapirija/Development/Career/WB/tmz-qa/app/.gauge/screenshots
08-03-2021 15:42:39.191 [Gauge] [DEBUG] Plugin screenshot is already installed.
08-03-2021 15:42:39.196 [Gauge] [DEBUG] Plugin reportserver is already installed.
08-03-2021 15:42:39.199 [Gauge] [DEBUG] Plugin python is already installed.
08-03-2021 15:42:39.202 [Gauge] [DEBUG] Plugin html-report is already installed.
08-03-2021 15:42:39.205 [Gauge] [DEBUG] Plugin xml-report is already installed.
08-03-2021 15:42:39.208 [Gauge] [DEBUG] Plugin spectacle is already installed.
08-03-2021 15:42:39.214 [Gauge] [DEBUG] Plugin flash is already installed.
08-03-2021 15:42:39.220 [Gauge] [DEBUG] Parsing started.
08-03-2021 15:42:39.221 [Gauge] [DEBUG] Started concepts parsing.
08-03-2021 15:42:39.946 [Gauge] [DEBUG] 0 concepts parsing completed.
08-03-2021 15:42:39.977 [Gauge] [DEBUG] Started specifications parsing.
08-03-2021 15:42:40.134 [Gauge] [DEBUG] 178 specifications parsing completed.
08-03-2021 15:42:40.134 [Gauge] [DEBUG] Applying tags filter: tmz-share & \!analytics
08-03-2021 15:42:40.159 [Gauge] [DEBUG] The following specifications satisfy filter criteria:
08-03-2021 15:42:40.159 [Gauge] [DEBUG] specs/projects/share/tmz/share_tmz__articles.spec
08-03-2021 15:42:40.159 [Gauge] [DEBUG] specs/projects/share/tmz/share_tmz__videos.spec
08-03-2021 15:42:40.159 [Gauge] [DEBUG] Parsing completed.
08-03-2021 15:42:41.891 [Gauge] [DEBUG] Checking updates...
08-03-2021 15:42:41.896 [Gauge] [DEBUG] Downloading https://downloads.gauge.org/plugin/flash?l=python&p=flash,html-report,python,reportserver,screenshot,spectacle,xml-report&o=darwin&a=amd64
08-03-2021 15:42:42.023 [Gauge] [DEBUG] Downloading https://downloads.gauge.org/plugin/html-report?l=python&p=flash,html-report,python,reportserver,screenshot,spectacle,xml-report&o=darwin&a=amd64
08-03-2021 15:42:42.081 [Gauge] [DEBUG] Downloading https://downloads.gauge.org/plugin/python?l=python&p=flash,html-report,python,reportserver,screenshot,spectacle,xml-report&o=darwin&a=amd64
08-03-2021 15:42:42.151 [Gauge] [DEBUG] Downloading https://downloads.gauge.org/plugin/reportserver?l=python&p=flash,html-report,python,reportserver,screenshot,spectacle,xml-report&o=darwin&a=amd64
08-03-2021 15:42:42.183 [Gauge] [DEBUG] Downloading https://downloads.gauge.org/plugin/screenshot?l=python&p=flash,html-report,python,reportserver,screenshot,spectacle,xml-report&o=darwin&a=amd64
08-03-2021 15:42:42.222 [Gauge] [DEBUG] Downloading https://downloads.gauge.org/plugin/spectacle?l=python&p=flash,html-report,python,reportserver,screenshot,spectacle,xml-report&o=darwin&a=amd64
08-03-2021 15:42:42.270 [Gauge] [DEBUG] Downloading https://downloads.gauge.org/plugin/xml-report?l=python&p=flash,html-report,python,reportserver,screenshot,spectacle,xml-report&o=darwin&a=amd64
08-03-2021 15:42:44.956 [python] [INFO] Python: 3.9.1
08-03-2021 15:42:44.956 [python] [DEBUG] Loading step implementations from /Users/xapirija/Development/Career/WB/tmz-qa/app/step_impl dirs.
08-03-2021 15:43:10.217 [Gauge] [CRITICAL] Failed to start gauge API: Timed out connecting to python

Expected behavior
Should execute test suite

Versions: 1.1.6

gauge -v
OS [e.g. MacOS 10.15.6 (19G2021) please be specific ] macOS 10.15.7
Runner info (Please specify the correct version of Java/Node/Python/Dotnet/Ruby) Python 3.9.1
IDE info (If using VS Code or Intellij IDEA) VS Code
Additional context`

**Reply by 2vireddy** [reply]
I see you are using a older version of gauge, maybe upgrading to the latest version might help you resolve the issue.
https://github.com/getgauge/gauge/releases/tag/v1.1.7

Also try downgrading the python version to 3.8 if above didn't work, since I am using gauge v1.7.7 with python 3.8 on windows
